### PR TITLE
Add build links

### PR DESCRIPTION
**Pull request by boshek**
@jonkeane this might be worth having your eyes on this prior to merging. Attached is the rendered report.

Changes:
 - Add additional columns that point to exact builds that failed

[crossbow-nightly-report.html.zip](https://github.com/ursacomputing/crossbow/files/8931618/crossbow-nightly-report.html.zip)

* apologies for the ugly diff. I ran styler on this.

### CODE REVIEW

**Review by assignUser — APPROVED** [review]
This change is awesome, thank you! 

Not necessarily for this PR but the "Error Log"-tabset is growing a bit out of control 😆 Maybe some kind of selector or drop down would be good?

File: crossbow-nightly-report.Rmd [hunk 1]
```diff
@@ -30,6 +32,54 @@ arrow_commit_links <- function(sha) {
 make_nice_names <- function(x) {
   toTitleCase(gsub("_", " ", names(x)))
 }
+
+arrow_build_table <- function(nightly_data, x, y) {
```
[the remark this answers is not in the document]
Comment (boshek) [reply crossbow-nightly-report.Rmd]: totally agree

**Comment by boshek** [comment]
> "Error Log"-tabset is growing a bit out of control

I replaced date with month and then added date a column in the tab. That should help a bit.

**Comment by boshek** [comment]
Thanks for the great feedback. Attached a re-rendered report here for posterity 
[crossbow-nightly-report.html.zip](https://github.com/ursacomputing/crossbow/files/8950121/crossbow-nightly-report.html.zip)

**Pull request merged**